packages/taquito/src/conversionUtils.ts

```diff
@@ -0,0 +1,9 @@
+import { PreparedOperation } from './prepare';
```

Comment:
I think we should put toForge in prepare provider instead, similar to toDryRun

Reply:
this has been converted to a method on the prepare provider